examples/default/Makefile

```diff
@@ -37,29 +37,24 @@ USEMODULE += ps
 # include and auto-initialize all available sensors
 USEMODULE += saul_default
 
-BOARD_PROVIDES_NETIF := acd52832 airfy-beacon b-l072z-lrwan1 cc2538dk fox \
-        iotlab-m3 iotlab-a8-m3 lsn50 mulle microbit native nrf51dk \
-        nrf51dongle nrf52dk nrf52840dk nrf52840-mdk nrf6310 \
-        openmote-cc2538 pba-d-01-kw2x remote-pa remote-reva samr21-xpro \
-        spark-core telosb yunjia-nrf51822 z1
-
-ifneq (,$(filter $(BOARD),$(BOARD_PROVIDES_NETIF)))
```

Comment:
Discussing this @fjmolinas offline and this is changing the previous behaviour: this application is not built for boards not providing a netif which was the case previously. So we still need `BOARD_PROVIDES_NETIF` here...